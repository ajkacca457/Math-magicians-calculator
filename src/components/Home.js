import React from 'react';

const Home = () => (
  <div className="home" style={{ width: '80%', margin: '10% auto' }}>
    <h1 style={{
      textAlign: 'left', fontSize: '140%', textDecoration: 'underline', color: 'orange',
    }}
    >
      Welcome to our page!
    </h1>
    <p style={{ textAlign: 'justify', lineHeight: '1.8' }}>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit,
      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
      aliquip ex ea commodo consequat.
      Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
      dolore eu fugiat nulla pariatur.
      Excepteur sint occaecat cupidatat non proident,
      sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>

    <p style={{ textAlign: 'justify', lineHeight: '1.8' }}>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit,
      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
      aliquip ex ea commodo consequat.
      Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
      dolore eu fugiat nulla pariatur.
      Excepteur sint occaecat cupidatat non proident,
      sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
    </p>
  </div>
);

export default Home;
