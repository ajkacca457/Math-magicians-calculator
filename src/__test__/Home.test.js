import React from 'react';
import TestRenderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../components/Home';

describe('Home Component', () => {

  it('renders correctly', () => {
    const hometree = TestRenderer
      .create(<Home />)
      .toJSON();
    expect(hometree).toMatchSnapshot();;
  });

  test('renders the title text of the Home page', () => {
    const { getByText } = render(<Home />);
    const title = getByText('Welcome to our page!');
    expect(title).toBeInTheDocument();
  });
});
