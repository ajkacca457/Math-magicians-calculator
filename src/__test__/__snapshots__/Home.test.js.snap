// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders correctly 1`] = `
<div
  className="home"
  style={
    Object {
      "margin": "10% auto",
      "width": "80%",
    }
  }
>
  <h1
    style={
      Object {
        "color": "orange",
        "fontSize": "140%",
        "textAlign": "left",
        "textDecoration": "underline",
      }
    }
  >
    Welcome to our page!
  </h1>
  <p
    style={
      Object {
        "lineHeight": "1.8",
        "textAlign": "justify",
      }
    }
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>
  <p
    style={
      Object {
        "lineHeight": "1.8",
        "textAlign": "justify",
      }
    }
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>
</div>
`;
